packages/e2e/cypress/e2e/api/sqlRunner.cy.ts

```diff
@@ -83,6 +88,178 @@ Object.entries(warehouseConnections).forEach(
+                                        JSON.parse(line),
+                                    );
+
+                                    expect(results).to.have.length(2);
+                                    expect(
+                                        results[0].payment_id.value.raw,
+                                    ).to.be.eq(1);
+                                    expect(
+                                        results[0].payment_id.value.formatted,
+                                    ).to.be.eq('1');
+                                    expect(
+                                        results[0].payment_method.value.raw,
+                                    ).to.be.eq('credit_card');
+                                    expect(
+                                        results[0].payment_method.value
+                                            .formatted,
+                                    ).to.be.eq('credit_card');
+                                    // TODO FIX amount, DIfferent warehouses have different format
+                                    // expect(results[0].amount.value.raw).to.be.eq("10.0000000000000000");
+
+                                    expect(
+                                        results[1].payment_id.value.raw,
+                                    ).to.be.eq(2);
+                                    expect(
+                                        results[1].payment_method.value.raw,
+                                    ).to.be.eq('credit_card');
+                                });
+                            } // Else keep polling
+                            else if (retries < maxRetries) {
+                                poll(retries + 1);
+                            } else {
+                                expect(
+                                    resp.body.results.status,
+                                    'Reached max number of retries without getting completed job',
+                                ).to.eq('completed');
+                            }
+                        });
+                    };
+                    poll();
+                });
+            });
         });
     },
 );
+
+// Load testing
+describe.skip(`Load testing`, () => {
+    beforeEach(() => {
+        cy.login();
+    });
+
+    it(`Load testing streaming 1M results `, () => {
+        // This SQL generates 50k random results on Postgres
+        const sql = `
+            SELECT
+                gs.id AS payment_id,
+                (1000 + random() * 10000)::int AS amount,  -- Random amount between 1000 and 11000
+                (current_date - (random() * 365)::int)::date AS payment_date,  -- Random date within the last year
+                md5(random()::text) AS payment_method,  -- Random payment reference
+                (random() * 100000)::int AS customer_id  -- Random customer_id between 0 and 100000
+            FROM
+                generate_series(1, 50000) AS gs(id)  -- Generate 50k rows
```

Comment:
Note that I'm about to enforce a limit of 5k on the SQL runner results. 
Enforce a limit in the SQL runner [#10238](https://github.com/lightdash/lightdash/issues/10238)